drivers/stepper/gpio_stepper_controller.c

```diff
@@ -326,6 +345,7 @@ static int gpio_stepper_motor_controller_init(const struct device *dev)
 		ARRAY_SIZE(gpio_stepper_motor_control_pins_##child) == 4,                          \
 		"gpio_stepper_controller driver currently supports only 4 wire configuration");    \
 	static const struct gpio_stepper_config gpio_stepper_config_##child = {                    \
+		.invert_direction = DT_PROP(child, invert_direction),                              \
```

Comment:
```suggestion
		.invert_direction = DT_PROP_OR(child, invert_direction, 0),                              \
```

Reply:
Boolean properties are true if present. They should not have a value. A boolean property is only false if it is completely missing in the DTS.